flask_ripozo/dispatcher.py

```diff
@@ -44,11 +46,10 @@ def exception_handler(dispatcher, accepted_mimetypes, exc):
     :return: A flask Response object.
     :rtype: flask.Response
     """
-    if isinstance(exc, RestException):
```

Comment:
This introduces a security risk of leaking unnecessary exception information.  If you want to override this you should pass your own exception handler function to the dispatcher.  Example:

    app = Flask('app')
    
    def my_error_handler(dispatcher, accepted_mimetypes, exc):
        adapter_klass = dispatcher.get_adapter_for_type(accepted_mimetypes)
        response, content_type, status_code = adapter_klass.format_exception(exc)
        return Response(response=response, content_type=content_type, status=status)
    
    dispatcher = FlaskDispatcher(app, error_handler=my_error_handler)

This commit should be rolled back.